lib/Runtime/ByteCode/ByteCodeSerializer.cpp

```diff
@@ -1182,6 +1192,8 @@ class ByteCodeBufferBuilder
                 DEFAULT_LAYOUT_WITH_ONEBYTE(Class);
                 DEFAULT_LAYOUT_WITH_ONEBYTE(ElementU);
                 DEFAULT_LAYOUT_WITH_ONEBYTE(ElementRootU);
+                DEFAULT_LAYOUT_WITH_ONEBYTE(ElementScopedC);
+                DEFAULT_LAYOUT_WITH_ONEBYTE(ElementScopedC2);
```

Comment:
I think these 2 may not be necessary, as they are only for eval, which should not hit in real world (this was hitting under -force:EvalCompile). so maybe better fix is to disable this flag in case of serialization flag (or visa-versa).